lib/php-version.spec.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const { test, given } = require('sazerac');
+const {
+  isHhvm,
+  versionReduction
+} = require('./php-version');
+
+describe('Text PHP version', function() {
+  test(isHhvm, () => {
+    given('hhvm').expect(true);
+    given('hhvm-3.18').expect(true);
+    given('7.1').expect(false);
+  });
+
+  test(versionReduction, () => {
+    given(['5.3', '5.4', '5.5']).expect('5.3 - 5.5');
+    given(['5.4', '5.5', '5.6', '7.0', '7.1']).expect('5.4 - 7.1');
+    given(['5.5', '5.6', '7.0', '7.1', '7.2']).expect('>= 5.5');
+    given(['5.5', '5.6', '7.1', '7.2']).expect('5.5, 5.6, 7.1, 7.2');
+    given(['7.0', '7.1', '7.2']).expect('>= 7');
+    given(['5.0', '5.1', '5.2', '5.3', '5.4', '5.5', '5.6', '7.0', '7.1', '7.2']).expect('>= 5');
+    given(['7.1', '7.2']).expect('>= 7.1');
+    given(['7.1']).expect('7.1');
+    given(['8.1']).expect('');
```

Comment:
This is a pretty confusing result. Is there a way to implement this that doesn't require us to hard-code the latest version of php? Or somewhere we can pull the available versions? There's a function called `regularUpdate`, which is a nice way to fetch pieces of information e.g. daily.

Reply:
We need to know the number of the latest version of PHP to make: `>= 5.6` or `>= 7`

Tell me more about `regularUpdate`

Reply:
It fetches a web resource, processes it, and caches the response. Here's an example usage, where it fetches a large JSON file, but only once every 4 hours:

https://github.com/badges/shields/blob/0d3d33bf1bfb7570eb6070c9d2cd2b1bc07451d2/server.js#L4586-L4594

There was some talk about it in that PR: #1158.

Reply:
It will be difficult to get the latest release number.
I can get a list of releases, but they go in the order of the release date, not the numbers.
* http://php.net/releases/
* http://php.net/releases/?json&version=7&max=10
* https://github.com/php/php-src/releases

Reply:
You could use the function we have in [lib/version.js](https://github.com/badges/shields/blob/master/lib/version.js#L14)!

Reply:
This is understandable.
I can take for example 10 latest versions, bring them to a general view and find the largest.
This will work with the release of 7.2, 7.3, 7.4, etc. But it will break after a major version changes.
This algorithm will not allow to determine the number of the last minor version in 7.x.
Need accumulate a list of versions in the local storage.

Reply:
I have an idea how to solve the problem. Later i will redo this.

Reply:
I reed list versions from GitHub
https://api.github.com/repos/php/php-src/git/refs/tags

Reply:
Can i use `regularUpdate()` in tests?